### DISCUSSION TITLE
Add HuggingFace

### DISCUSSION THREAD

**Discussion by Pablohn26** [react-icons/react-icons#828] [New Icon Set]
I would like to add HuggingFace icon. 

You can find it [here](https://huggingface.co/brand)

## What is HuggingFace

Hugging Face is the collaboration platform for the machine learning community.

The Hugging Face Hub works as a central place where anyone can share, explore, discover, and experiment with open-source ML. HF empowers the next generation of machine learning engineers, scientists, and end users to learn, collaborate and share their work to build an open and ethical AI future together.

With the fast-growing community, some of the most used open-source ML libraries and tools, and a talented science team exploring the edge of tech, Hugging Face is at the heart of the AI revolution.

**Comment by extergeist** [comment]
I hope they add it eventually
